justfile

```diff
@@ -0,0 +1,155 @@
+export RUSTFLAGS := "-D warnings"
+export RUSTDOCFLAGS := "-D warnings"
+
+
+
+[private]
+default:
+	just -l --unsorted
```

Comment:
Would it make sense to bind the default recipe to a more frequently used command? Something like "I always want to run this before I commit".